corehq/apps/domain/decorators.py

```diff
@@ -225,6 +234,30 @@ def _inner(request, *args, **kwargs):
     return _inner
 
 
+def check_lockout(fn):
+    @wraps(fn)
+    def _inner(request, *args, **kwargs):
+        username = _get_username_from_request(request)
+        user = CouchUser.get_by_username(username)
+        if user and user.is_web_user() and user.is_locked_out():
+            return HttpResponse(json.dumps({"error": "maximum password attempts exceeded"}),
```

Comment:
There is `json_response` util method (in dimagi-utils) which can cut down lines here.

Also, in case this is user-facing error, it should be i18nized